kolibri/core/tasks/api.py

```diff
@@ -230,38 +240,64 @@ def validate_create_req_data(self, request):
                 )
 
             # Check permissions the DRF way
```

Comment:
Maybe move this comment inside the function where you moved the code too?

Reply:
No, not a great idea. That function has its own doc string that defines what its doing. 

Let this comment stay here for clarity of what's going on.

Reply:
My note was mostly just about the fact that the 'DRF way' refers to what is happening now inside the function that is being called.